src/s3/s3_handler.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+
+from botocore.exceptions import ClientError
+
+from s3.s3_client import connect_client
+from s3.settings import AWSSettings
+
+BUCKET_NAME = AWSSettings.BUCKET_NAME
+
+
+async def download_file_from_s3(file_uuid):
+    try:
+        async with await connect_client() as client:
+            response = await client.get_object(
+                Bucket=BUCKET_NAME, Key=f"{file_uuid}.jpg"
```

Comment:
utilize this function in a view + no need for the `BUCKET_NAME` constant + `/downloads` goes to the settings

Reply:
https://stackoverflow.com/questions/55873174/how-do-i-return-an-image-in-fastapi